src/data/builder-tools.js

```diff
@@ -762,6 +762,14 @@ export const Showcases = [
     getstarted: null,
     tags: ["operatortool", "cli"],
   },
+  {
+    title: "Plutip",
+    description: "Cardano tool that aims to help dApp developers with integration testing and contracts debugging using disposable private network",
+    preview: require("./builder-tools/plutip.png"),
+    website: "https://github.com/mlabs-haskell/plutip",
+    getstarted: "https://github.com/mlabs-haskell/plutip/blob/master/README.md#tutorials",
```

Comment:
```suggestion
    getstarted: null,
```

There is a specific section in get started on developer portal dedicated to builder tools get started tutorials. The format is in markdown. Feel free to add your guide there as well, otherwise please write null in that field.

Reply:
Will it be possible to add "getting started" later?
We are still stabilizing our API, so I put `null` for now.

Reply:
yes, we can approve this entry now & you can submit another PR to add the `getstarted:` link whenever you're ready 🤓

Reply:
Great!
I've put `null` there with latest commit.

Thank you for review!